### ISSUE TITLE
Start video playback in fullscreen landscape view by default

### ISSUE THREAD

**Issue by seanthegeek** [nextcloud/android#9454]
### Is your feature request related to a problem? Please describe.
I usually keep auto-rotation turned of on my Android phone, and lock it to a portrait orientation to prevent annoying accidental rotation if I'm reading in bed or holding my phone out to show someone something.

Currently, video playback in the Nextcloud app remains in portrait mode until I enable auto-rotate on my phone, then rotate my phone, and turn auto rotate back off when I'm done with the video.  

Having to always use the three dot menu to make the video full screen is annoying too.

### Describe the solution you'd like
Apps from video streaming services like Netflix, Hulu, and Disney+ override the orientation lock and play videos in landscape orientation and full screen once a user taps play. The Nextcloud app should do the same.

**Comment by AlvaroBrey** [comment]
I don't think this is a good idea. Most apps which are not dedicated video players, for example Telegram, and even others which are (like YouTube) open the video horizontally first. I think automatic fullscreen would be a source of confusion for users.

In https://github.com/nextcloud/android/pull/9482, we added a fullscreen toggle to the video controls so that the three dot menu is not required. Hopefully this will make entering fullscreen easier enough.

**Comment by XueSheng-GIT** [comment]
@AlvaroBrey: I would confirm that it is quite easy to change to fullscreen (as you said, just press the toggle). But toggling to fullscreen and especially rotating the device does invoke a restart of the video somehow and results in "double playback" from time to time (video sound runs in background and is not related to the video controls anymore... app must be killed to stop playback).

**Comment by Rirmach** [comment]
I can't agree more. Maybe just an extra "button" to switch to vertical display without auto-rotation on?
